cloud-ts-url-shortener-cache-http/cache.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as cloud from "@pulumi/cloud";
+import * as config from "./config";
+
+// A simple cache abstraction that wraps Redis.
+export class Cache {
+    public readonly get: (key: string) => Promise<string>;
+    public readonly set: (key: string, value: string) => Promise<void>;
```

Comment:
Funny! Just today I was looking at whether we can merge https://github.com/pulumi/examples/pull/51. (The answer appears to be "yes.") Any chance we can use the new style and maybe even merge that PR?

Reply:
I'll see if that can be done.

Reply:
Done.